testutil/ciao-down/workload.go

```diff
@@ -76,6 +81,40 @@ func (wkld *workload) save(ws *workspace) error {
 }
 
 func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
+	if err == nil {
+		switch u.Scheme {
+		case "http", "https":
+			workloadFile, err := ioutil.TempFile("/tmp", ".workload")
```

Comment:
this file will be closed by ```getFile``` 
https://github.com/01org/ciao/blob/master/testutil/ciao-down/download.go#L77